collector/lib/NetworkStatusNotifier.cpp

```diff
@@ -326,15 +330,25 @@ void NetworkStatusNotifier::AddConnections(::google::protobuf::RepeatedPtrField<
       // the case where we send a close event for a connection that it doesn't
       // know about.
       //
-      if (!rate_limiter.Allow(delta_entry.first.container())) {
-        CLOG_THROTTLED(INFO, std::chrono::seconds(5)) << "Rate limiting connections reported from container " << delta_entry.first.container();
+      const std::string& container = delta_entry.first.container();
```

Comment:
Could we use a `std::string_view` directly? I believe it's a trivial cast back to `const std::string&` so the call to `Allow` on the next line should still work.
```suggestion
      const std::string_view container = delta_entry.first.container();
```